lib/livebook_web/live/home_live/import_file_upload_component.ex

```diff
@@ -81,13 +81,10 @@ defmodule LivebookWeb.HomeLive.ImportFileUploadComponent do
 
   @impl Phoenix.LiveComponent
   def handle_event("save", _params, socket) do
```

Comment:
Can you please revert the change to this function? Both ways are equivalent but this change is unrelated to this PR, thank you. :)

Reply:
I even think the non-tuple return value has been deprecated :)